file-formats/ddls/examples/z_aff_example_ddls.ddls.objectdependencies.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Formatting the json would increase the readability

Reply:
Let's handle this in a separate PR.

Reply:
> Formatting the (.objectdependencies.)json would increase the readability.
Let's merge this PR and work on this in https://github.com/SAP/abap-file-formats/issues/163